contributors/join-us/recruiting.md

```diff
@@ -19,57 +19,127 @@ If you're considering a role at Mattermost, here are the **Top 10 Reasons to wor
 9. **Learn, Master, Teach:** Building new skills, becoming experts, and training others is core to our culture.
 10. **Friends around the world:** Build relationships with staff and contributors in dozens of countries.
 
-## Interview Stages
+In addition to this page, we also encourage anyone interested in interviewing at Mattermost to read [About Mattermost](https://handbook.mattermost.com/company/about-mattermost). 
+
+## Working and Meeting Across the Globe
+
+### Work Anywhere
+
+We’re headquartered in Palo Alto, California and, like many open source companies, a large portion of Mattermost, Inc. works across the globe. We run as a distributed, high performance and high trust team using a few principles:
+
+#### 1. Work Anywhere with Good Internet
+
+Working from anywhere means more time with loved ones, no commute, and controlling everything about your workspace.
+
+The one requirement is having the ability to occassionally [join video calls using Zoom](https://support.zoom.us/hc/en-us/articles/201362023-System-Requirements-for-PC-Mac-and-Linux) between 8am and 1pm California time, which requires a good enough internet connection.
+
+Recommended download/upload speeds should be 8 Mbps/512 kbps for smooth group calling and screensharing, with a bare minimum of 4 Mbps/128 kbps. The [Ookla Speed Test](https://www.speedtest.net/) can be used to check your internet speed.
+
+There are exceptions to this requirement, for example: part-time roles that don’t need to take meetings, or roles providing support outside core meeting hours. Hiring Managers and Recruiters will indicate any time zone requirements for specific roles and/or teams.
+
+We are continually expanding where we can hire staff, updated lists can be found [here](https://handbook.mattermost.com/contributors/join-us).
+
+#### 2. Be Explicit
+
+Working in a distributed organization means it may take time before a person reads your message and responds, so make sure you are clear.
+
+Communicate relevant information in the first message:
+
+* Avoid “Are you free to talk?” and “Can we talk about something at 3pm?”, instead use “Can we speak about ulimit issue Customer X is having?”
+
+Write unambiguously:
+
+* Avoid “I think the answer is Yes”, instead use “Yes”, “No”, or “I think the answer is Yes, I’ll confirm with Steve and let you know.”
+
+Likewise, we emphasize written communication. Reading is faster than listening and it reaches more people, which means more happens in less time.
+
+#### 3. Follow Through
+
+When working on problems, make sure to be clear about next steps, and either follow through or create a ticket to track the work.
 
-We can hire from many countries and regions across the globe. While in-office interviews typically go through 1 - 3 interview rounds, our candidates and interviewers may be located in various timezones and therefore each interview is scheduled separately. The stages are as follows:
+#### 4. Hang Out
```

Comment:
```suggestion
#### 4. Make Meaningful Connections
```

Reply:
What're your thoughts on "Build Camraderie" as a tie-back to 'quality work relationships" and staff enablement?

Reply:
I like the concept. I wonder how well "Build Camraderie" as a term is known by non-English audiences, something to consider

Reply:
That's a good point. Do you think "Embrace Camaraderie" would make a difference, or is it the word 'Camaraderie'? Curious, are there any terms used in other regions that you, @jasonblais @justinegeffen know of that have a similar concept? I like the idea of using a non-English, or even non-North American word or term.

Reply:
@natalie-hub wondering if you were open to reaching out with someone like Jesús? He might be able to share what it means to him.

We could even simplify it to "Get to know each other", something GitLab uses at https://about.gitlab.com/handbook/values/#collaboration (may need to scroll down a little bit)

Reply:
@jasonblais @justinegeffen  -  sorry for the roundabout, "Make Meaningful Connections" makes sense and is aligned with the newly created Workplace V2MOM, see link. https://docs.google.com/document/d/1pB63Iqo0gXRpzKOm7ugqeFkUgLlOa_H61djcFaid9Vc/edit?userstoinvite=amy.nicol%40mattermost.com&ts=5f59058d&actionButton=1  CC: @hannaparks